frontend/src/metabase/common/components/EntityPicker/components/EntityPickerModal/EntityPickerModal.tsx

```diff
@@ -347,6 +348,8 @@ export function EntityPickerModal<
 
   const titleId = useUniqueId("entity-picker-modal-title-");
 
+  const modalRef = useRef<HTMLElement>(null);
```

Comment:
How is this used?

Maybe the name "modalContentRef" would also work, to be extra specific, to distinguish it from the modal's root

Reply:
Oh good catch, I meant to remove this. My bad.